### PR TITLE
Add devFriendly name to the SchemaNode

### PR DESCRIPTION
**Pull request by SHeimlich**

### CODE REVIEW

File: src/tools/schema2graph.ts [hunk 1]
```diff
@@ -70,8 +70,26 @@ export class SchemaNode {
   // This currently assumes a top-level schema can be found for every connection.
   // Note: This will change once we enable handle connections with tuples.
   static entityTypeForConnection(connection: HandleConnectionSpec, nodes: SchemaNode[]): string {
```
Comment (piotrswigon) [inline src/tools/schema2graph.ts:72]: Can we remove this one now?

File: src/tools/schema2kotlin.ts [hunk 2]
```diff
@@ -246,7 +246,8 @@ abstract class Abstract${particle.name} : ${this.opts.wasm ? 'WasmParticleImpl'
     for (const connection of particle.connections) {
       const handleName = connection.name;
       const entityType = SchemaNode.entityTypeForConnection(connection, nodes);
-      const handleInterfaceType = this.handleInterfaceType(connection, entityType);
+      const devFriendlyName = SchemaNode.devFriendlyEntityTypeForConnection(connection, nodes);
```
Comment (piotrswigon) [inline src/tools/schema2kotlin.ts:249]: the variable name could still be entityType I think, the fact that you want to use the devFriendlyName is encapsulated by the method name used.

File: src/tools/schema2graph.ts (hunk 1 above)
Comment (SHeimlich) [reply src/tools/schema2graph.ts]: Done

File: src/tools/schema2kotlin.ts (hunk 2 above)
Comment (SHeimlich) [reply src/tools/schema2kotlin.ts]: Done

**Pull request merged**